package.json

```diff
@@ -3,18 +3,14 @@
   "displayName": "Bazel Jest Runner",
   "description": "Run Jest test with Bazel in VSCode",
   "version": "0.1.0",
-  "files": [
-    "out",
-    "images"
-  ],
-  "publisher": "mscolnick",
+  "publisher": "TumuGuskun",
```

Comment:
Got it. So `mscolnick`'s package is the base from which we're building our vscode extension.

Have we double checked the licensing here? I'd feel a bit icky if we were stepping on this person's intellectual property.

Reply:
hmm I assumed there was an MIT license but you're right there's no license there. I can reach out to them/put more creds in